packages/amplication-git-utils/src/git/github.service.ts

```diff
@@ -90,7 +90,7 @@ export class GithubService {
       url: repo.html_url,
       private: repo.private,
       fullName: repo.full_name,
-      admin: repo.permissions.admin,
+      admin: repo.permissions?.admin || false,
```

Comment:
why do we need the `|| false` part?

Reply:
because permissions can be undefined so we need to provide bool